### PR TITLE
WNB-125:Forgot password screens

### PR DESCRIPTION
**Pull request by vitali-sakalou**
This is a "Forgot password screens". The process of sending a new password is temporarily replaced by using setTimeout().

### CODE REVIEW

**Comment by vitali-sakalou** [comment]
No need to merge. Needs some work)

**Comment by ghost** [comment]
Ok, don't forget to request reviews from me and Dima once you're done

**Comment by vitali-sakalou** [comment]
This is WNB-125:Forgot password screens - **refactoring**. Implemented on the same way with obtaining data for organizations. Please give me some notes and feedback.

File: src/components/PasswordRecovery/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { Grid, Container } from 'semantic-ui-react';
+import { Header, ContinueButton, ContinueInput } from './styles';
+
+const PasswordRecovery = props => {
```
Comment (ghost) [inline src/components/PasswordRecovery/index.js:5]: could be const PasswordRecovery = { cbFunction } => {
  ...
}

File: src/components/PasswordRecovery/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    props.cbFunction(email);
+  };
+
+  return (
+    <Grid stackable columns="1">
+      <Grid.Row stretched columns="1">
+        <Grid.Column>
+          <Container text textAlign="center">
+            <Header as="h1">
+              Для восстановления пароля введите адрес вашей электронной почты
```
Comment (ghost) [inline src/components/PasswordRecovery/index.js:18]: Do we address our users formally ("введите" или "введи")?

Comment (ghost) [reply src/components/PasswordRecovery/index.js:18]: Btw we could differentiate users by their status (man, young man etc...) and use different pronouns, but that's for the future, definitely

File: src/components/PasswordRecovery/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { Grid, Container } from 'semantic-ui-react';
+import { Header, ContinueButton, ContinueInput } from './styles';
+
+const PasswordRecovery = props => {
+  const [email, setEmail] = useState('');
+
+  const handleContinue = () => {
+    props.cbFunction(email);
```
Comment (ghost) [inline src/components/PasswordRecovery/index.js:9]: Maybe we should add props validation

File: src/routes/ForgotPassword/Component.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    []
+  );
+
+  const cbSendNewPassword = email => {
+    const { sendPassword } = props;
+    sendPassword(email);
+  };
+
+  const { loading, loaded, error } = props;
```
Comment (ghost) [inline src/routes/ForgotPassword/Component.js:20]: Please display some error message if there's an error

File: src/components/PasswordRecovery/index.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            </Header>
+            <ContinueInput
+              placeholder={'Адрес эл. почты*'}
+              type={'email'}
+              value={email}
+              onChange={e => {
+                setEmail(e.target.value);
+              }}
+            />
+            <ContinueButton size="large" secondary onClick={handleContinue}>
```
Comment (ghost) [inline src/components/PasswordRecovery/index.js:28]: Please add some very basic validation for email (presence of @)

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Refactor handleSuccess and handleError logic in our reducer factory: it now correctly handles strings, arrays and objects

**Review by ikushlianski — APPROVED** [review]

**Review by letutsenik — APPROVED** [review]

**Pull request closed without merging**